lib/ansible/config/base.yml

```diff
@@ -652,17 +653,16 @@ DEFAULT_GATHERING:
   default: 'implicit'
   description:
     - This setting controls the default policy of fact gathering (facts discovered about remote systems).
-    - "When 'implicit' (the default), the cache plugin will be ignored and facts will be gathered per play unless 'gather_facts: False' is set."
-    - "When 'explicit' the inverse is true, facts will not be gathered unless directly requested in the play."
-    - "The 'smart' value means each new host that has no facts discovered will be scanned,
-      but if the same host is addressed in multiple plays it will not be contacted again in the playbook run."
     - "This option can be useful for those wishing to save fact gathering time. Both 'smart' and 'explicit' will use the cache plugin."
   env: [{name: ANSIBLE_GATHERING}]
   ini:
     - key: gathering
       section: defaults
   version_added: "1.6"
-  choices: ['smart', 'explicit', 'implicit']
+  choices:
+    implicit: the cache plugin will be ignored and facts will be gathered per play unless 'gather_facts: False' is set
+    explicit: facts will not be gathered unless directly requested in the play
+    smart: each new host that has no facts discovered will be scanned, but if the same host is addressed in multiple plays it will not be contacted again in the run
```

Comment:
```suggestion
    smart: each new host that has no facts discovered will be scanned, but if the same host is addressed in multiple plays it will not be contacted again in the run.
```